lab_1/main.py

```diff
@@ -69,7 +116,26 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    if not isinstance(tokens, list) or not isinstance(word, str):
+        return []
+    elif not isinstance(right_context_size, int) and not (left_context_size, int):
+        return []
+    else:
```

Comment:
Shorten the function by one line. Please, explain how and why it can be shortened.